src/GafferArnoldUI/DecayVisualiser.cpp

```diff
@@ -34,7 +34,9 @@
 //
 //////////////////////////////////////////////////////////////////////////
 
-#include "GafferSceneUI/LightFilterVisualiser.h"
+#include "GafferSceneUI/Export.h"
```

Comment:
I think this one is redundant too.

Reply:
https://github.com/GafferHQ/gaffer/commit/e4be79c410ad38b91127b9d2120aab37fa134c06